src/main/java/io/orten/nano/impl/UserService.java

```diff
@@ -0,0 +1,70 @@
+package io.orten.nano.impl;
+
+import io.orten.nano.model.User;
+import io.orten.nano.util.Database;
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserService {
+
+        public static List<User> u_list = new ArrayList<User>();

```

Comment:
Give it a more meaningful camelCased name

Reply:
Replaced u_list by userList